scripts/pylib/twister/twisterlib/environment.py

```diff
@@ -437,6 +437,10 @@ def add_parse_arguments(parser = None):
         help="Re-use the outdir before building. Will result in "
              "faster compilation since builds will be incremental.")
 
+    parser.add_argument(
+        "--no-path-name", action="store_true",
```

Comment:
may be we can put all those into a concise report mode, and combinet several opetion into one

Reply:
sorry, I don't follow. Which "all those"? I am adding a single arg which changes the way the suite names are created (not adding path). I don't think this option can be combined in one with others.

Reply:
what do you mean by "option"? A grouping for args? How would it make the interface simpler? What is an "internal control flag"? The arg I am adding can be used with any other argument. It not only change the internal handling of names but also affects the names in the output report, so to me it is not really an "internal" argument. What other arguments should be in this `concise-report`? Sorry, I really don't follow. If it is a general comment about idea on modification of twister interface, maybe you can open an RFC where show what you'd like to achieve?

Reply:
let's discuss this in weekly meeting. I mean maybe we can have some default report options:
--verbose_mode
--concise_mode

and in each mode  a flags dict is build in like
concise_mode = {
'short_path' => true,
'log_time' => false,
'log_in_line' => false,
}

and those flags in the dict are mapped  to twister internal control logic